docker-compose.yml

```diff
@@ -32,6 +32,56 @@ services:
   rosie:
     image: serenata/rosie
 
+  prometheus:
+    image: arthursens/serenata-prometheus
+    ports: 
+    - 9090:9090
+    links:
+    - node_exporter:node_exporter
+    - process_exporter:process_exporter
+    - postgres_exporter:postgres_exporter
+    - bi_exporter:bi_exporter
+
+  grafana:
+    image: arthursens/serenata-grafana
```

Comment:
arthursens/serenata-grafana was built from [this Dockerfile](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-933098848ceee5eb4c5202f8be105679R1)